docs/guides/dns.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'The "dns" gadget' weight: 10
+---
+
+The dns gadget prints information about DNS requests performed by the different
+pods.
+
+Create a `demo` a namespace:
+
+```
+$ kubectl create ns demo
+namespace/demo created
+```
+
+Start the dns gadget. Notice that it'll capture events on the `default`
+namespace as the `--namespace` parameter was not explicitly set.
```

Comment:
But we want it to capture events on the `demo` namespace, isn't it?

Reply:
oh, that's right.

Reply:
```
Start the dns gadget. Notice that it'll capture events on the `default`
namespace as the `--namespace` parameter was not explicitly set
```
In my opinion, this description should be updated as well because looks like you are going to launch it without the `--namespace` parameter to monitor the `default` namespace but you're not doing that anymore. Don't you think?

Reply:
Done.